libzen.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: libzen
+  version: 0.4.41
+  epoch: 0
+  description: "ZenLib is a C++ utility library"
+  copyright:
+    - license: Zlib
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates
+      - git
+      - build-base
+      - autoconf
+      - automake
+      - libtool
+      - pkgconf
+      - pkgconf-dev
+      - glibc
+      - wolfi-baselayout
+      - zlib-dev
+
+pipeline:
+  - name: Checkout ZenLib
+    uses: git-checkout
+    with:
+      repository: https://github.com/MediaArea/ZenLib.git
+      tag: v0.4.39
+      expected-commit: 13b9d3dbd2088390fbdccab32ae4560526389fbc
+
+  - name: Build and Install ZenLib
+    runs: |
+      pwd
+      ls -la
+      ./autogen.sh
+      ./configure --prefix=/usr
+      make
+      make DESTDIR="${{targets.destdir}}" install
+    working-directory: Project/GNU/Library
+
+subpackages:
+  - name: "libzen-dev"
+    description: "Development files for libzen"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.destdir}}/usr/share/doc/libzen"
+          cp -r ZenLib/Documentation/* "${{targets.destdir}}/usr/share/doc/libzen/" || true
+        working-directory: ZenLib/Project/GNU/Library
```

Comment:
This looks like a pipeline for installing a docs package, not a -dev package. For the `-dev`,  could you use the `split/dev` pipeline?